stm32l0_pwrtest.ino

```diff
@@ -169,10 +180,10 @@ void setup(void)
         setup_flash();
 
 #ifdef ARDUINO_MCCI_CATENA_4610 || ARDUINO_MCCI_CATENA_4611 || \
-	\ ARDUINO_MCCI_CATENA_4612 || ARDUINO_MCCI_CATENA_4618
+        \ ARDUINO_MCCI_CATENA_4612 || ARDUINO_MCCI_CATENA_4618
```

Comment:
What's the backslash doing at the front of old 172/new 183?  I think that's some kind of typo.

Reply:
Removed the backslashes at front of second line